### PR TITLE
feat: make all adapters available on all platforms

### PR DESCRIPTION
**Pull request by KnorpelSenf**
This PQ makes all frameworks available on all platforms by no longer splitting them up between files.

This has a number of benefits:
- greatly simplifies the framework setup
- no longer makes any assumptions about where adapters exist and where they don't, as many web frameworks actually get ported between the various runtimes
- removes a cyclic dependency between files

@kraftwerk28 can you review this?

### CODE REVIEW

**Review by dcdunkan — APPROVED** [review]
LGTM.

**Review by roj1512 — APPROVED** [review]
Stamp-rubber LGTM.

**Comment by KnorpelSenf** [comment]
@dcdunkan @roj1512 can you share which adapters you've tested?

File: src/convenience/frameworks.ts [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+                });
+                resolveResponse(res);
+            }
+        },
+        handlerReturn: new Promise((resolve) => {
+            resolveResponse = resolve;
+        }),
+    };
+};
+/** Native CloudFlare workers (service worker) */
```
Comment (KnorpelSenf) [inline src/convenience/frameworks.ts:261]:
```suggestion

/** Native CloudFlare workers (service worker) */
```

**Comment by KnorpelSenf** [comment]
Did other manual testing, merging.

**Comment by dcdunkan** [comment]
> @dcdunkan @roj1512 can you share which adapters you've tested?

Uhh... none. I just _LGTM_-ed the changes because they looked good to me :D

**Pull request merged**